README.md

```diff
@@ -283,16 +283,54 @@ resend: {
 
 ### Community Products API
 
-The library provides functions for interacting with Community Products.
+Streamr client provides functions for working with Community Products. All are async, that is, they return a Promise.
+
+#### Admin functions
+
+Name | Returns | Description
+---- | ------- | -----------
+deployCommunity() | Transaction | Deploy a new community product
+createSecret(communityAddress, secret[, name]) | Create a secret for a community product
+communityIsReady(address) | | Wait until community is operated
+addMembers(communityAddress, memberAddressList) | | Add members
+kick(communityAddress, memberAddressList) | | Kick members out from community
+```
```

Comment:
perhaps  use \`\`\`js for syntax-highlighted code snippets?
i.e.

```js
const community = await client.deployCommunity()
community.address           // already available before deployment
await community.deployed()  // waits until contract is deployed
await community.isReady()   // waits until community is operated
```

instead of

```
const community = await client.deployCommunity()
community.address           // already available before deployment
await community.deployed()  // waits until contract is deployed
await community.isReady()   // waits until community is operated
```
```

Reply:
changed the rest, too